index.scss

```diff
@@ -37,7 +37,8 @@
 
 .slider-decorator-0,
 .slider-decorator-1 {
-  position: static !important;
+  //position: static !important;
```

Comment:
I believe it is intentional, but I will confirm and remove it altogether if that is the case. Thanks!